docs/1.getting-started/8.server.md

```diff
@@ -0,0 +1,86 @@
+---
+navigation.icon: uil:laptop-connection
+description: Build full-stack applications, fetch data from your database, create APIs, or even generate static server-side content like a sitemap or a RSS feed, from a single codebase.
+---
+
+# Server
+
+Nuxt's server framework allows you to build **full-stack applications**. For example, you can fetch data from a database or another server, create an API or even generate static server-side content like a sitemap or an RSS feed - all from a single codebase.
+
+::alert{type="info"}
+For more information about the `~/server` directory, please visit the [server directory structure section](/docs/guide/directory-structure/server).
+::
+
+## Powered by Nitro
+
+Nuxt's server is [Nitro](https://github.com/unjs/nitro). Nitro was originally created for Nuxt but is now part of [UnJS](https://unjs.io/) and used for other frameworks - and can even be used on its own.
+
+Using Nitro give Nuxt superpowers:
+
+- Full control of the server-side part of your app
+- Deploy your app anywhere
+- Hybrid rendering for your app
+
+## Full control of the server-side part of your app
+
+With Nitro, you can easily manage the server-side of your Nuxt 3 app; from endpoints to middleware.
+
+Both endpoints and middleware can be defined like this:
+
+```ts [server/api/test.ts]
+export default defineEventHandler({
+  // ... Do whatever you want here
+})
+```
+
+And you can directly return `text`, `json`, `html` or even a `stream`.
+
+Out-of-the-box, Nitro supports **hot module replacement** and **auto-import** like the other parts of your Nuxt application.
+
+::alert{type="info"}
+Learn more about endpoints and middlewares in [the server directory structure section](/docs/guide/directory-structure/server).
+::
+
+## Deploy your app anywhere
+
+Nitro offers the ability to deploy your Nuxt 3 anywhere, from a bare metal server to the edge network, with a start time near to a few milliseconds. That's fast!
+
+Today, Nitro offers more than 15 presets to build your Nuxt 3 app for different cloud providers and servers, including:
+
+- [Cloudflare Workers](https://workers.cloudflare.com/)
+- [Netlify Functions](https://www.netlify.com/products/functions/)
+- [Vercel Edge Network](https://vercel.com/docs/edge-network/introduction)
+
+Or for other runtimes:
+
+- [Deno](https://deno.land/)
+- [Bun](https://bun.sh/)
+
+::alert{type="info"}
+Learn more about deployment in [the related section](https://nuxt.com/docs/getting-started/deployment).
+::
+
+## Hybrid rendering for your app
+
+Need for both static and dynamic pages in your Nuxt 3 app? Nitro covers your back!
+
+Nitro has a powerful feature called `routeRules` which allow you to define a set of rules to customize the routes rendering mode of your Nuxt 3 app.
```

Comment:
```suggestion
Need for both static and dynamic pages in your Nuxt app? Nitro covers your back!

Nitro has a powerful feature called `routeRules` which allows you to define a set of rules to customize how each route of your Nuxt app is rendered (and more).
```

Reply:
I'd even change the start to 
"Need both static and..."

Reply:
Not an expert enough to know! 🤷‍♂️